lgc/builder/BuilderImpl.cpp

```diff
@@ -81,6 +81,12 @@ bool BuilderImplBase::supportDpp() const {
   return getPipelineState()->getTargetInfo().getGfxIpVersion().major >= 8;
 }
 
+// =====================================================================================================================
```

Comment:
We need to get rid of these "support*" functions and express them instead as target features in TargetInfo. But adding this one here for now is fine. :-)

Reply:
FWIW, I'm of different minds on this one.

On the one hand, it makes things more orthogonal and separated out. On the other hand, it causes us to forget what each hardware supports.

Reply:
A better way is to have all features encapsulated in target info class and target info provides a bunch of query functions (based on GFX IP) to get whatever we want to know.